### PR TITLE
Fix bug in cljr--resolve-alias

### PR DESCRIPTION
**Pull request by luxbock**
- The old regexp was too greedy.

### CODE REVIEW

**Comment by expez** [comment]
How did this manifest?  Would you mind adding a test to ensure we don't break it again when we refactor?

**Comment by luxbock** [comment]
I was trying to use the new `cljr-add-stubs` command in a namespace where I had one ns aliased as `s` and `clojure.string` before it as `str`, and it picked up the wrong symbol to use. 

Sure I'll add a test-case a bit later.

File: clj-refactor.el [hunk 1]
```diff
@@ -522,7 +522,7 @@ e.g. `re-search-forward'"
   (save-excursion
     (cljr--goto-ns)
     (when (re-search-forward
-           (format ":as\\s-+%s" (regexp-quote alias))
+           (format ":as\\s-+s\\>" (regexp-quote alias))
```
Comment (Malabarba) [inline clj-refactor.el]: Shouldn't this be `":as\\s-+%s\\_>"` (note the % and the _)

Comment (luxbock) [inline clj-refactor.el]: Whops! Fixed.

**Comment by expez** [comment]
Thanks for adding a test!  Squash the commits and we should be good to go :+1:

**Comment by expez** [comment]
Thanks! :+1:

**Pull request merged**